### PR TITLE
chore(step-sequence): remove classic theme support - FE-3123/FE-3124

### PR DESCRIPTION
**Pull request by mkrds**
### Proposed behaviour
This PR removes classic theme support from StepSequence and StepSequenceItem components
This PR also refactors old stories to new CSF/MDX formats.

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

File: cypress/features/regression/stepSequence.feature [hunk 1]
```diff
@@ -3,7 +3,7 @@ Feature: Step Sequence component
 
   @positive
   Scenario Outline: I set orientation to <orientation>
-    When I open default "Step Sequence" component in noIFrame with "stepSequence" json from "commonComponents" using "<nameOfObject>" object name
+    When I open step-sequence "Step Sequence Test" component in noIFrame with "stepSequence" json from "commonComponents" using "<nameOfObject>" object name
```
Comment (DlgSHi) [inline cypress/features/regression/stepSequence.feature:6]: this file needs to be removed as we have covered these tests by `chromatic`, also needs to be removed used objects from the  `cypress/fixtures/commonComponents/stepSequence.json`

File: cypress/features/regression/stepSequenceItem.feature [hunk 2]
```diff
@@ -15,8 +15,8 @@ Feature: Step Sequence Item component
 
   @positive
   Scenario Outline: I set status to <status>
-    When I open default "Step Sequence Item" component in noIFrame with "stepSequence" json from "commonComponents" using "<nameOfObject>" object name
-    Then text "Step Label" color is set to "<color>"
+    When I open step-sequence-item "Step Sequence Test" component in noIFrame with "stepSequence" json from "commonComponents" using "<nameOfObject>" object name
```
Comment (DlgSHi) [inline cypress/features/regression/stepSequenceItem.feature:18]: this TC needs to be removed, as we are checking colours in `chromatic` also delete please the corresponding json objects from `cypress/fixtures/commonComponents/stepSequence.json`

File: cypress/support/step-definitions/step-sequence-steps.js [hunk 3]
```diff
@@ -22,6 +23,10 @@ Then('hidden label is set to {word}', (hiddenLabel) => {
   ariaLabel(ARIA_LABEL).contains(hiddenLabel).should('have.text', hiddenLabel);
 });
 
+Then('label {string} color is set to {string}', (text, color) => {
```
Comment (DlgSHi) [inline cypress/support/step-definitions/step-sequence-steps.js:26]: and remove please also `orientation is set to {string}` it is not used anymore :)

**Comment by DlgSHi** [comment]
First of all this branch needs to be updated with `master` and after that also `accessibility` tests need to be updated

**Review by grabkowski — APPROVED** [review]

**Review by ianforster-sage — DISMISSED** [review]

**Review by grabkowski — APPROVED** [review]

**Review by grabkowski — DISMISSED** [review]

**Review by ianforster-sage — DISMISSED** [review]

**Review by ianforster-sage — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by grabkowski — APPROVED** [review]

**Pull request merged**